### DISCUSSION TITLE
cloudwatch metrics of a ASG filtered by tag

### DISCUSSION THREAD

**Discussion by MartinEtcheverry** [grafana/grafana#64917] [Dashboards, Panels & Visualizations]
Hi, I'm pretty new to this question, and I don't know if this is the right place to ask. But I have some stacks that I redeploy regularly, they create ASGs with similar names every time I redeploy. I tag the ASGs with the same tag for each stack. The point is that I want to compare the values of the previous ASG with the current one. I thought about doing a filter by tag of the ASGs, but when I make the variable in the dashboard "dimension_values($region, AWS/AutoScaling, GroupTotalInstances, AutoScalingGroupName)" I don't see how to add the Tag to be able to filter using it.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I think this is more of a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
